src/apps/xpubs/xpubs.py

```diff
@@ -307,6 +317,33 @@ async def save_menu(self, show_screen):
             return self.export_coldcard
         return None
 
+    async def export_multiple_accounts_xpubs(self, from_account, to_account, file_format, show_screen):
```

Comment:
Looks like `file_format` is not used here. Makes sense to remove the `save_menu` screen above, or implement saving in json format as well.

Reply:
You are right, I will remove it for now.